members/natalie_hooven.md

```diff
@@ -0,0 +1,33 @@
+--
+Layout: default
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/example_student.png" alt="Placeholder Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Natalie Hooven (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: I am an undergraduate researcher currently focusing on modeling macromolecules.
+
+<!-- Name at least one research topic amongst this list -->
+**Things I'm excited about right now**: Methane!
+
+<!-- Ultimately, we'll use this section to
+     include papers and talks, and contributions
+     But for now put whatever you want -->
+**Cool stuff I've done**: I helped build this website, and I crochet some of my own clothes for fun too.
+
+
+<!-- If you have photos you would like to exhibit,
+     save them as `/assets/member_images/your_name_photo_#.png`
+     and replace example_student below -->
+
+!--|      |      |      |--
+!--|:----:|:----:|:----:|--
+!--|![](/website/assets/img/example_student_1.png) | ![](/website/assets/img/example_student_2.png) | ![](/website/assets/img/example_student_3.png) | --
+
```

Comment:
Make sure to delete this if you aren't adding additional images -- you can either use markdown commenting (start with `<!--` and end with `-->`) or delete these lines altogether.